### PR TITLE
Add AVMetadataObjectTypeInterleaved2of5Code

### PR DESCRIPTION
**Pull request by skaman**
I had troubles to read some kind of ITF barcodes, i'm not 100% sure but i think the type Interleave 2 of 5 is an ITF variant. Anyway adding it I fixed the read of my ITF barcodes. I think this fix can solve a couple of opened issues about these kind of barcodes.

### CODE REVIEW

**Comment by Klathmon** [comment]
Yes, Interleave 2 of 5 is actually what "ITF" stands for, and this will fix the problem of ITF not being readable if they aren't 14 characters long.

This PR fixes issues #455, #354, and #341

File: src/ios/CDVBarcodeScanner.mm [hunk 1]
```diff
@@ -623,6 +624,7 @@ - (NSString*)formatStringFromMetadata:(AVMetadataMachineReadableCodeObject*)form
     if (format.type == AVMetadataObjectTypeCode128Code)     return @"CODE_128";
     if (format.type == AVMetadataObjectTypeCode93Code)      return @"CODE_93";
     if (format.type == AVMetadataObjectTypeCode39Code)      return @"CODE_39";
+    if (format.type == AVMetadataObjectTypeInterleaved2of5Code) return @"ITF";
     if (format.type == AVMetadataObjectTypeITF14Code)          return @"ITF";
```
Comment (jcesarmobile) [inline src/ios/CDVBarcodeScanner.mm]: Now "ITF" is twice here.
I think yours is fine, but the old one should be changed to "ITF_14".
Can you change that and also fix the merge conflicts?

**Comment by jcesarmobile** [comment]
Closing in favor of #636

**Comment by herparis** [comment]
how can i make it to work on android?
i installed this plugin, i want to read the barcode interleaved 2 to 5 but it doesnt do anything wenn i aim with the camera, my cordova version is 8.0 and my cordova-android version is 7.0

**Pull request closed without merging**